src/types.ts

```diff
@@ -384,6 +385,26 @@ export interface EarlyAccessFeatureResponse {
     earlyAccessFeatures: EarlyAccessFeature[]
 }
 
+/** A user's support tickets */
+export interface SupportTicket {
+    // Sync this with the backend's SupportTicketSerializer!
+    id: number
+    created_at: Date
+    updated_at: Date
+    subject: string
+    description: string
+    status: 'open' | 'closed'
```

Comment:
infoMessage: string <- that can be stuff like 
- "We're currently at an offsite, expect delayed responses" 
- "We're having an incident" 
- "You're in the queue at 15th place"
- "Expect to hear back from us within 72h" <- what the time here is will be configured in the api side based on priority, where the exact message is configured in the posthog app